/*      */ package com.sybase.jdbc3.jdbc.resource;
/*      */ 
/*      */ import java.util.ListResourceBundle;
/*      */ 
/*      */ public class Messages extends ListResourceBundle
/*      */ {
/*   28 */   private static final Object[][] CONTENTS = { { "JZ001", "User name property '%1s' too long. Maximum length is 30." }, { "JZ002", "Password property '%1s' too long. Maximum length is 30." }, { "JZ0NE", "Incorrect URL format. URL: %1s. Error message: %2s" }, { "JZ003", "Incorrect URL format. URL: %1s" }, { "JZ0PN", "Specified port number of %1s was out of range. Port numbers must meet the following conditions: 0 <= portNumber <= 65535" }, { "JZ0BC", " [extra arg%1s:%2s]\n" }, { "JZ0BM", " \n Also found an internal error:  Could not find argument marker in error message.  \n This does not affect normal operation of the application, but Sybase Technical Support should be made aware of this error.\n" }, { "JZ004", "User name property missing in DriverManager.getConnection(..., Properties)." }, { "JZ006", "Caught IOException: %1s" }, { "JZ0NC", "wasNull called without a preceding call to get a column." }, { "JZ008", "Invalid column index value %1s." }, { "JZ00F", "Cursor property cannot be changed after the cursor has been declared." }, { "JZ00E", "You cannot execute a cursor statement while the cursor is open.  Close the statement first." }, { "JZ0LC", "You cannot call the %1s method on a ResultSet which is using a language cursor to fetch rows. Try setting the LANGUAGE_CURSOR connection property to false." }, { "JZ00G", "No column values were set for this row update." }, { "JZ00H", "The result set is not updatable.  Remove the 'FOR READ ONLY' clause from the query" }, { "JZ0RM", "The %1s method may not be called after updateRow or deleteRow." }, { "JZ0RD", "You cannot call any of the ResultSet.get* methods on a row that has been deleted with the deleteRow() method." }, { "JZ0IR", "getXXX may not be called on a column after it has been updated in the result set with a java.io.Reader." }, { "JZ0IS", "getXXXStream may not be called on a column after it has been updated in the result set." }, { "JZ0BD", "Out of range or invalid value used for method parameter." }, { "JZ0BR", "The cursor is not positioned on a row that supports the %1s method." }, { "JZ0BT", "The %1s method is not supported for ResultSets of type %2s." }, { "010RC", "The requested ResultSet type and concurrency is not supported. They have been converted." }, { "000D3", "Your Sybase JDBC license will expire soon. Please obtain a new license. It will expire on %1s." }, { "JZ0D4", "Unrecognized protocol in Sybase JDBC URL: %1s." }, { "JZ0D5", "Error loading protocol %1s." }, { "JZ0D6", "Unrecognized version number %1s specified in setVersion. Choose one of the SybDriver.VERSION_* values, and make sure that the version of jConnect that you are using is at or beyond the version you specify." }, { "JZ0HC", "Illegal character '%1s' encountered while parsing hexadecimal number." }, { "JZ009", "Error encountered in conversion.  Error message: %1s" }, { "JZ00A", "Invalid precision and scale specified for a numeric value." }, { "JZ00C", "The precision and scale specified cannot accomodate numeric value '%1s'." }, { "JZ00I", "Invalid scale. The specified scale must be >= 0." }, { "JZ0TE", "Attempted conversion between an illegal pair of types. Valid database datatypes are: '%1s'" }, { "JZ0TC", "Attempted conversion between an illegal pair of types." }, { "JZ0TI", "jConnect cannot make a meaningful conversion between the database's type of '%1s' and the requested type of '%2s'." }, { "JZ0I3", "Incorrect argument %1s passed to method %2s. \nVerify in the product documentation or in the JDBC API that the correct argument was passed. " }, { "JZ014", "You may not setTransactionIsolation(Connection.TRANSACTION_NONE). \nThis level cannot be set; it can only be returned by a server." }, { "JZ00B", "Numeric overflow." }, { "JZ0C0", "Connection is already closed." }, { "JZ0C1", "An IOException occurred which closed the connection." }, { "JZ0S1", "Statement state machine: Trying to FETCH on IDLE statement." }, { "JZ0S2", "Statement object has already been closed." }, { "JZ0R0", "ResultSet has already been closed." }, { "JZ0R1", "Result set is IDLE as you are not currently accessing a row." }, { "JZ0R2", "No result set for this query." }, { "JZ0R5", "The ResultSet is currently positioned beyond the last row. You cannot perform a get* operation to read data in this state." }, { "S0022", "Invalid column name '%1s'." }, { "JZ0R3", "Column is DEAD.  This is an internal error; please report it to Sybase technical support." }, { "JZ0R4", "Column does not have a text pointer. It is not a text/image column or the column is NULL." }, { "JZ0P1", "Unexpected result type." }, { "JZ0P4", "Protocol error. This message indicates an internal product problem.  Report this error to Sybase Technical Support." }, { "JZ0I5", "An unrecognized CHARSET property was specified: %1s." }, { "JZ0I6", "An error occured converting UNICODE to the charset used by the server.  Error message: %1s" }, { "JZ0IB", "The server's default charset of %1s does not map to an encoding that is available in the client Java environment. Because jConnect will not be able to do client-side conversion, the connection is unusable and is being closed. Try using a later Java version, or try including your Java installation's i18n.jar or charsets.jar file in the classpath." }, { "JZ0TD", "Caught ThreadDeath." }, { "JZ0I7", "No response from proxy gateway." }, { "JZ0I8", "Proxy gateway connection refused.  Gateway response: %1s" }, { "JZ0IA", "Truncation error." }, { "JZ0TS", "Truncation error trying to send %1s." }, { "JZ0I9", "This InputStream was closed." }, { "JZ0S8", "An escape sequence in a SQL Query was malformed: '%1s'." }, { "JZ0SH", "A static function escape was used, but the metadata accessor information was not found on this server." }, { "JZ0SI", "A static function escape %1s was used which is not supported by this server." }, { "JZ0SJ", "Metadata accessor information was not found on this database. Please install the required tables as mentioned in the jConnect documentation." }, { "010UF", "Attempt to execute use database command failed.  Error message: %1s" }, { "010DF", "Attempt to set database at login failed.  Error message: %1s" }, { "010MX", "Metadata accessor information was not found on this database. Please install the required tables as mentioned in the jConnect documentation. Error encountered while attempting to retrieve metadata information: %1s" }, { "010SJ", "Metadata accessor information was not found on this database. Please install the required tables as mentioned in the jConnect documentation." }, { "JZ0SA", "Prepared Statement: Input parameter not set, index: %1s." }, { "JZ0SB", "Parameter index out of range: %1s." }, { "JZ0S3", "The inherited method %1s cannot be used in this subclass." }, { "JZ0SC", "Callable Statement: attempt to set the return status as an input parameter." }, { "01S09", "The local transaction method %1s cannot be used while a global transaction is active on this connection." }, { "01S10", "The local transaction method %1s cannot be used on a pre-System 12 XAConnection." }, { "JZ0SD", "No registered parameter found for output parameter." }, { "JZ0SE", "Invalid object type (or null object) specified for setObject()." }, { "JZ0ST", "jConnect cannot send a Java object as a literal parameter in a query. Make sure that your database server supports Java objects and that the LITERAL_PARAMS connection property is set to false when you execute this query." }, { "JZ0SF", "No Parameters expected. Has query been sent?" }, { "JZ0SG", "A CallableStatement did not return as many output parameters as the application had registered for it." }, { "JZ0SU", "A Date or Timestamp parameter was set with a year of %1s, but the server can only support year values between %2s and %3s. If you're trying to send data to date or timestamp columns or parameters on Adaptive Server Anywhere, you may wish to send your data as Strings, and let the server convert them." }, { "JZ0P7", "Column is not cached; use RE-READABLE_COLUMNS property." }, { "JZ0S4", "Cannot execute an empty (zero-length) query." }, { "JZ0SM", "jConnect could not execute a stored procedure because there was a problem sending the parameter(s). This problem was likely caused because the server does not support a specific datatype, or because jConnect did not request support for that datatype at connect time.\nTry setting the JCONNECT_VERSION connection property to a higher value. Or, if possible, try sending your procedure execution command as a language statement." }, { "JZ0SL", "Unsupported SQL type %1s." }, { "JZ0SN", "setMaxFieldSize: field size cannot be negative." }, { "JZ0SR", "setMaxRows: max rows cannot be negative." }, { "JZ0SS", "setQueryTimeout: query timeout time cannot be negative." }, { "ZZ00A", "The method %1s has not been completed and should not be called." }, { "JZ0NS", "The method %1s is not supported and should not be called." }, { "010AF", "SEVERE WARNING: An assertion has failed, please use devclasses to determine the source of this serious bug. Message = %1s" }, { "JZ0AF", "ASSERTION: [%1s] on class %2s failed in thread %3s." }, { "JZ0P8", "The RSMDA Column Type Name you requested is unknown.  This is a Sybase internal error; please report it to technical support." }, { "010P4", "An output parameter was received and ignored." }, { "010P6", "A row was received and ignored." }, { "JZ011", "Format exception encountered while parsing connection property %1s." }, { "JZ015", "Illegal value set for GSSMANAGER_CLASS connection property. The property value must be a String or an Object that extends org.ietf.jgss.GSSManager." }, { "JZ012", "Internal Error. Please report it to Sybase technical support. Wrong access type for connection property %1s." }, { "010UP", "Unrecognized connection property %1s ignored." }, { "JZ0J0", "Offset and/or length values exceed the actual text/image length." }, { "010DP", "Duplicate connection property %1s ignored." }, { "010SK", "Database cannot set connection option %1s." }, { "JZ0PA", "The query has been cancelled and the response discarded.  The cancel was probably issued by another statement on the connection." }, { "JZ0T2", "Listener thread read error.  Check network communications." }, { "JZ0EM", "End of data." }, { "JZ0T7", "Listener thread read error -- caught ThreadDeath.  Check network communications." }, { "JZ0T8", "Data received for an unknown request.Please report this error to Sybase Technical Support." }, { "JZ0TO", "Read operation timed out." }, { "JZ0T4", "Write operation timed out.  Timeout in milliseconds: %1s." }, { "JZ0T5", "Cache is full. Use default or larger value for STREAM_CACHE_SIZE." }, { "JZ0T6", "Error reading Tunnelled TDS URL." }, { "JZ0H0", "Unable to start thread for event handler; event name = %1s." }, { "JZ0H1", "An event notification was received but the event handler was not found; event name = %1s." }, { "010SL", "Out-of-date metadata accessor information was found on this database.  Ask your database administrator to load the latest scripts." }, { "JZ0SK", "The oj escape is not supported for this type of database server.  Workaround: use server-specific outer join syntax, if supported.  Consult server documentation." }, { "JZ00L", "Login failed.  Examine the SQLWarnings chained to this exception for the reason(s)." }, { "JZ00M", "Login timed out. Check that your database server is running on the host and port number you specified. Also, check the database server for other conditions (such as a full tempdb) that might be causing it to hang." }, { "010HD", "Sybase high-availability failover is not supported by this type of database server." }, { "JZ0F1", "Sybase high-availability failover connection was requested but the companion server address is missing." }, { "JZ0F2", "Sybase high-availability failover has occurred. The current transaction is aborted, but the connection is still usable. Retry your transaction." }, { "JZ0F3", "HADR failover has occurred. The connection is now re-established to the new primary. The application will need to re-establish the context" }, { "010HA", "The server denied your request to use the high-availability feature.  Please reconfigure your database, or do not request a high-availability session." }, { "0100V", "The version of TDS protocol being used is too old.  Version: %1s.%2s.%3s.%4s" }, { "010HT", "Hostname property truncated, maximum length is 30: " }, { "010PO", "Property LITERAL_PARAM has been reset to 'false' because DYNAMIC_PREPARE was set to 'true'." }, { "010SP", "File could not be opened for writing.  File: %1s.  Error message: %2s" }, { "010TP", "The connection's initial character set, %1s, could not be converted by the server. The server's proposed character set, %2s, will be used, with conversions performed by jConnect." }, { "010TQ", "jConnect could not determine the server's default character set. This is likely because of a metadata problem.\nPlease install the required tables as mentioned in the jConnect documentation.\nThe connection is defaulting to the ascii_7 character set, which can handle only characters in the range 0x00 through 0x7F." }, { "010SN", "Permission to write to file was denied.  File: %1s.  Error message: %2s" }, { "010SM", "This database does not support the initial proposed set of capabilities, retrying." }, { "JZ010", "Unable to deserialize an Object value.  Error text: %1s" }, { "JZ0PB", "The server does not support a requested operation." }, { "010SQ", "The connection or login was refused, retrying connection with the next host/port address." }, { "JZ0D7", "Error loading url provider %1s.  Error message: %2s" }, { "JZ013", "Error obtaining JNDI entry: %1s. Error message: %2s" }, { "JZ019", "Error obtaining SERVERNAME : %1s" }, { "JZ021", "The Specified %1s file not found." }, { "JZ022", "The Specified %1s file has an unknown format." }, { "JZ024", "The Specified Server : %1s has no entry in the interfaces/sql.ini file :%2s." }, { "JZ027", "The Specified SECMECH entrys: %1s for Server : %2s in interfaces/sql.ini file :%3s are not Supported." }, { "JZ026", "The Specified Protocol : %1s for Server : %2s in interfaces/sql.ini file :%3s is not Supported." }, { "JZ0NF", "Unable to load an instance of com.sybase.jdbcx.SybSocketFactory.  Verify the SYBSOCKET_FACTORY property to insure that you have spelled the class name correctly, that the package is fully specified, that the class is available in your class path, and that it has a public zero-argument constructor." }, { "JZ0US", " The SYBSOCKET_FACTORY connection property was set, and the PROXY connection property was set to the URL of a servlet.  The jConnect driver does not support this combination.  If you want to send secure HTTP from an applet running within a browser, use a proxy URL beginning 'https://'." }, { "JZ0BI", " setFetchSize: The fetch size should be set with the following restrictions -- 0 <= rows <= (maximum number of rows in the ResultSet)." }, { "JZ0BJ", "The value set for the IMPLICIT_CURSOR_FETCH_SIZE connection property must be > 0." }, { "JZ0PC", "The number and size of parameters in your query require widetable support. But either the server does not offer such support, or it was not requested during the login sequence. Try setting the JCONNECT_VERSION property to >= 6 if you wish to request widetable support." }, { "JZ0PD", "The size of the query in your dynamic prepare is large enough that you require widetable support. But either the server does not offer such support, or it was not requested during the login sequence. Try setting the JCONNECT_VERSION property to >= 6 if you wish to request widetable support." }, { "JZ0PE", "The number of columns in your cursor declaration OR the size of your cursor declaration itself are large enough that you require widetable support. But either the server does not offer such support, or it was not requested during the login sequence. Try setting the JCONNECT_VERSION property to >= 6 if you wish to request widetable support." }, { "JZ0BI", " setFetchSize: The fetch size should be set with the following restrictions -- 0 <= rows <= (maximum number of rows in the ResultSet)." }, { "JZ0NF", "Unable to load an instance of com.sybase.jdbcx.SybSocketFactory.  Verify the SYBSOCKET_FACTORY property to insure that you have spelled the class name correctly, that the package is fully specified, that the class is available in your class path, and that it has a public zero-argument constructor." }, { "JZ0SO", "Invalid ResultSet concurrency type: %1s" }, { "JZ0SP", "Invalid ResultSet type: %1s" }, { "JZ0SW", "Invalid ResultSet holdability type: %1s" }, { "JZ0SQ", "Invalid UDT type: %1s" }, { "JZ0BS", "Batch Statements are not supported" }, { "JZ0BE", "BatchUpdateException: Error occurred while executing batch statement: %1s" }, { "JZ0BP", "Output Parameters are not allowed in Batch Update Statements" }, { "010PF", "One or more jars specified in the PRELOAD_JARS connection property could not be loaded." }, { "JZ0XS", "The server does not support XA-style transactions.  Please verify that the transaction feature is enabled and licensed on this server." }, { "JZ0XC", "%1s is an unrecognized transaction coordinator type." }, { "JZ0XU", "Current user does not have permission to do XA-style transactions.  Be sure user has %1s role." }, { "JZ0CL", "You must define the CLASS_LOADER property when using the PRELOAD_JARS property." }, { "JZ0GS", "A Generic Security Services API exception occurred. The major error code is %1s.\nThe major error message is: %2s\nThe minor error code is %3s.\nThe minor error message is: %4s" }, { "JZ0GN", "Error instantiating the class %1s as a GSSManager. The exception was %2s. Please check your CLASSPATH and make sure the GSSMANAGER_CLASS property value refers to a fully qualified class name of a GSSManager implementation." }, { "JZ0GC", "Error casting a %1s as a GSSManager. Please check the value you are setting for the GSSMANAGER_CLASS connection property. The value must be a String that specifies the fully qualified class name of a GSSManager implementation. Or, it must be an Object that extends org.ietf.jgss.GSSManager." }, { "010HN", "The client did not specify a SERVICE_PRINCIPAL_NAME Connection property. Therefore, jConnect is using the hostname of %1s as the service principal name." }, { "010KF", "The server rejected your Kerberos login attempt. Most likely, this was because of a GSS (Generic Security Services) exception. Please check your Kerberos environment and configuration." }, { "01S07", "Adaptive Server may round or truncate nanosecond values" }, { "010CP", "AutoCommit option has changed to true. All pending statements on this transaction (if any) are committed." }, { "JZ0SV", "Combination of setting parameters by name and by index is not allowed in the same CallableStatement." }, { "01S08", "This connection has been enlisted in a Global Transaction. All pending statements on the current local Transactions(if any) have been rolled back." }, { "JZ017", "Savepoint is not valid." }, { "JZ018", "This method can not be applied to this type of savepoint." }, { "JZ0MD", "ResultSet metadata is not available." }, { "JZ0FP", "Incorrect value passed for parameter %1s." }, { "JZ0GK", "The %1s array can not be null and has to contain only one key." }, { "JZ0NK", "Generated keys are not available because either the Statement.NO_GENERATED_KEYS was used or no keys were automatically generated." }, { "JZ028", "Illegal value set for JCE_PROVIDER_CLASS connection property. The property value must be a fully qualified provider class name passed as a String or an instance of java.security.Provider." }, { "JZ029", "Error looking up address for ALTERNATE_SERVER_NAME %1s, (%2s)." }, { "JZ0LA", "Failed to instantiate Cipher object. Transformation %1s is not implemented by any of the loaded JCE providers." }, { "JZBK1", "SybBCP class is NOT initialized Please Re-Run MDA sqls to update the MDA stored procedures." }, { "01ZZZ", "Password has expired Please set the NEWPASSWORD property with the new password or use sp_password to change passwords." }, { "JZBK3", "Bulk load table does not exists." }, { "JZNNA", "Column does not allow null values" }, { "JZBKI", "Invalid value set for property ENABLE_BULK_LOAD" }, { "JZBK4", "Illegal usage of sql statements mixed with batches in bcp/arrayinsert mode" }, { "JZBK5", "autocommit should be set to true when running bulk load in bcp mode" }, { "JZBK6", "ASE 15.7 or latter and 'allow wide dol rows' DB option must be enabled to insert rows with offsets greater than 8191." }, { "JZ030", "The method %1s is not supported." }, { "JZ031", "Failed to unwrap the object of %1s." }, { "01S11", "WARNING: Your data might be truncated." }, { "JZ033", "Unknown Blob type returned by the server" }, { "JZ036", "Reference to this Large Object [LOB] is no longer valid in database. Check if you have called free() or check if transaction ended." }, { "JZ037", "Value of offset/position/start should be in the range [1, len] where len is length of Large Object[LOB]." }, { "JZ038", "Length of the object should be >= 0." }, { "JZ039", "Length of the Stream/Reader should be >= -1." }, { "JZ040", "%1s operation failed. The %2s has been closed already." }, { "JZ041", "%1s operation failed on the %2s." }, { "JZBK7", "Failed to insert data. Total data size (%1s bytes) exceeds the maximum row size (%2s bytes)allowed for the table %3s." }, { "JZ042", "Large Object setters can not be mixed with other setters when ENABLE_LOB_LOCATOR and HOMOGENEOUS_BATCH are set to TRUE. java.sql.Types %1s was mixed with java.sql.Types %2s." }, { "01S12", "Unable to continue with HOMOGENEOUS_BATCH protocol, falling back to normal batching." }, { "JZ043", "LOB objects are not supported for any of the possible varients of ENABLE_BULK_LOAD property, but false. Please consider using alternate setter APIs to insert the data." }, { "JZ044", "Server-side locators can not be created within the batch if, SEND_BATCHPARAMS_IMMEDIATE is TRUE. Try using client side LOBs or set SEND_BATCHPARAMS_IMMEDIATE to FALSE." }, { "01S13", "Connected ASE server does not support the set option 'logbulkcopy' needed for logging BCP. Falling back to normal bulk load without logging which is equivalent to setting ENABLE_BULK_LOAD=BCP." }, { "JZ045", "Cannot add batch row [%1s]. Maximum batch size support for current configuration is %2s.\nTo use large batch size, increase INTERNAL_READ_BUFFER_LIMIT (if set) or the JVM heap memory." }, { "JZ046", "Invalid value set for property INTERNAL_READ_BUFFER_LIMIT. Valid unit are %/KB/MB/GB. Values specified in % is considered as percentage of total virtual memory size. Valid range is %1s to %2s(when KB is specified)." }, { "JZ00D", "getCursorName() called without a previous call to setCursorName()." }, { "JZ0T3", "Read operation timed out." } };
/*      */ 
/*      */   public Object[][] getContents()
/*      */   {
/* 1251 */     return CONTENTS;
/*      */   }
/*      */ }

/* Location:           C:\Users\zhangtx\Desktop\ribo\ribo.jar
 * Qualified Name:     com.sybase.jdbc3.jdbc.resource.Messages
 * JD-Core Version:    0.5.4
 */